errai-common/src/main/java/org/jboss/errai/common/client/dom/elemental2/Elemental2DomUtil.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.errai.common.client.dom.elemental2;
+
+import com.google.gwt.user.client.ui.RootPanel;
+import com.google.gwt.user.client.ui.Widget;
+import elemental2.dom.HTMLElement;
+import elemental2.dom.Node;
+import jsinterop.base.Js;
+
+/**
+ * Provides utility methods for interacting with the DOM.
+ * @author Guilherme Carreiro <ggomes@redhat.com>
```

Comment:
Our preferred _style_ is to not include ```@author``` tags.

Reply:
I think that Errai adopts a different code style.. Am I missing something?

Reply:
Probably I am!